packages/playground/src/components/user-management/form-user-reset-password.tsx

```diff
@@ -12,19 +12,7 @@ import {
   Button,
   Icon
 } from '@harnessio/canary'
-
-interface UsersProps {
-  uid: string
-  display_name?: string
-}
-
-interface FormResetPasswordrDialogProps {
-  user: UsersProps | null
-  onReset: () => void
-  onClose: () => void
-  isResetting: boolean
-  resetSuccess: boolean
-}
+import { FormResetPasswordrDialogProps } from './interfaces'
```

Comment:
spelling of `Password`.. there seems to be an extra `r`?

Reply:
fixed